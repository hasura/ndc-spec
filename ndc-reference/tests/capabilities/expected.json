{
  "version": "0.1.2",
  "capabilities": {
    "query": {
      "aggregates": {},
      "variables": {}
    },
    "mutation": {},
    "relationships": {
      "relation_comparisons": {},
      "order_by_aggregate": {}
    }
  }
}