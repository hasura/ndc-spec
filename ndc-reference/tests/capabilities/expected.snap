---
source: ndc-reference/bin/reference/main.rs
expression: response
---
{
  "version": "0.2.11",
  "capabilities": {
    "query": {
      "aggregates": {
        "filter_by": {},
        "group_by": {
          "filter": {},
          "order": {},
          "paginate": {}
        }
      },
      "variables": {},
      "nested_fields": {
        "filter_by": {
          "nested_arrays": {
            "contains": {},
            "is_empty": {}
          }
        },
        "order_by": {},
        "aggregates": {},
        "nested_collections": {}
      },
      "exists": {
        "named_scopes": {},
        "unrelated": {},
        "nested_collections": {},
        "nested_scalar_collections": {}
      }
    },
    "mutation": {},
    "relationships": {
      "relation_comparisons": {},
      "order_by_aggregate": {},
      "nested": {
        "array": {},
        "filtering": {},
        "ordering": {}
      }
    }
  }
}
